torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,41 @@
+import io
+
+import torch
+
+from typing import Union
+
+
+def dynamo_export(
+    torch_model: torch.nn.Module,
+    *args,
+    opset_version: Optional[int] = None,
+    export_options: Optional[ExportOptions] = None,
+    verbose: Optional[bool] = None,
+    **kwargs,
+) -> ExportOutput:
+    """Export a torch.nn.Module to a ONNX graph
+
+    Args:
+        torch_model: User model to be exported
+        args: Positional inputs to `torch_model`
+        opset_version: Target ONNX opset version.
+        export_options: Advanced export options
+        verbose: Print basic debug when True
+        kwargs: Keyword inputs to `torch_model`
+
+    Returns:
+        An in-memory representation of the exported ONNX model.
+        See ``ExportOutput`` for reference.
+    """
+
+    if not options:
+        options = DefaultExportOptions(opset_version=opset_version, verbose=verbose)
+
+
+class ExportOutput:
+    def __init__(self, output: onnx.ModelProto):
+        ...
+
+    def save(self, destination: Union[str, io.BytesIO]) -> None:
```

Comment:
just for discussion for chaining : `def save(self, destination: Union[str, io.BytesIO]) -> Self:`